manifests/client.pp

```diff
@@ -14,7 +14,9 @@
   $client_packages         = $qpid::client::params::client_packages,
 ) inherits qpid::client::params {
 
-  class { '::qpid::client::install': } ~>
-  class { '::qpid::client::config': }
+  include ::qpid::client::install
+  include ::qpid::client::config
+
+  Class['qpid::client::install'] ~> Class['qpid::client::config']
```

Comment:
I did not like the ~> on the newline so I changed it to this format. Maybe we should standardize on something.